test/sequential/sequencerStaking.sequencerJoining.test.ts

```diff
@@ -14,156 +14,193 @@ import {
   ExtrinsicResult,
   filterAndStringifyFirstEvent,
   getProvidingSeqStakeData,
+  waitSudoOperationFail,
 } from "../../utils/eventListeners";
 import { BN } from "@polkadot/util";
 import { getEventResultFromMangataTx } from "../../utils/txHandler";
 import { signTx } from "gasp-sdk";
 import { stringToBN } from "../../utils/utils";
 
 let chain: any;
-let testUser: User;
+let testUser1: User;
+let testUser2: User;
+let testUser3: User;
+let testUser4: User;
+let testUser5: User;
 let stakeAmount: BN;
+let sudoUser: User;
 
 beforeAll(async () => {
   await initApi();
   await setupApi();
   stakeAmount = (await SequencerStaking.minimalStakeAmount()).addn(1000);
-});
-
-beforeEach(async () => {
+  sudoUser = getSudoUser();
+  await SequencerStaking.removeAddedSequencers();
   //There shouldn't be any sequencer in activeSequencers
-  [testUser] = setupUsers();
+  [testUser1, testUser2, testUser3, testUser4, testUser5] = setupUsers();
   chain = "Ethereum";
-  await Sudo.batchAsSudoFinalized(Assets.mintNative(testUser));
-  testUser.addAsset(GASP_ASSET_ID);
+
+  await Sudo.batchAsSudoFinalized(
+    Assets.mintNative(testUser1),
+    Assets.mintNative(testUser2),
+    Assets.mintNative(testUser3),
+    Assets.mintNative(testUser4),
+    Assets.mintNative(testUser5),
+  );
+  testUser1.addAsset(GASP_ASSET_ID);
+  testUser2.addAsset(GASP_ASSET_ID);
+  testUser3.addAsset(GASP_ASSET_ID);
+  testUser4.addAsset(GASP_ASSET_ID);
+  testUser5.addAsset(GASP_ASSET_ID);
```

Comment:
why this change?

Reply:
I received a blinking error when I was working with these tests. They worked well separately, but 2 tests had Tx Map errors when I ran them together.